homeassistant/components/alarm_control_panel/manual.py

```diff
@@ -185,3 +187,13 @@ def _validate_code(self, code, state):
         if not check:
             _LOGGER.warning("Invalid code given for %s", state)
         return check
+
+    @property
+    def state_attributes(self):
```

Comment:
Overwrite `device_state_attributes` not `state_attributes`.

Reply:
What’s the difference between them?

Reply:
Accordng to the docstrings of the helper `Entity` class.
- `state_attributes` returns the state attributes and is implemented by component base class.
- `device_state_attributes` returns device specific state attributes and is implemented by platform classes.

`device_state_attributes` are merged with `state_attributes` before setting the state of the entity.